script.module.lwe_mapper/addon.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.lwe_mapper"
+       name="mapper"
+       provider-name="linuxwhatelse"
+       version="1.1.0">
+
+    <requires>
+        <import addon="xbmc.python" version="2.24.0"/>
+    </requires>
+
+    <extension library="lib" point="xbmc.python.module"/>
+
+    <extension point="xbmc.addon.metadata">
+        <language>en</language>
+        <platform>all</platform>
+        <summary lang="en">Simple URL-Scheme resolver</summary>
```

Comment:
Can you also change those from en to en_GB? See: https://github.com/xbmc/generator-kodi-addon/blob/master/generators/app/templates/addon.xml